net-vpn/vtun/vtun-3.0.3-r1.ebuild

```diff
@@ -56,4 +53,5 @@ src_install() {
 	newinitd "${FILESDIR}"/vtun.rc vtun
 	insinto etc
```

Comment:
`insinto /etc` is the style we prefer in general

Reply:
Sure, done